src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from azure.cli.core import AzCommandsLoader
+
+import azure.cli.command_modules.acs._help  # pylint: disable=unused-import
```

Comment:
Your namespace for an extension is now different and is under `azext_aks_preview`. 
You will need to modify all your imports to conform.

Reply:
I edited it to `import azext_aks_preview._help`, issue still persists

Reply:
Did you modify all your other imports as well? Also please continue to debug through the errors you are seeing until CI passes(Travis will likely give you the information you need to address your problems) and the command works as expected.